.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Let us know about an issue with DIM
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+**What version of DIM are you using?**
+Find the version in "About DIM" from the menu.
```

Comment:
(button with three bars, upper left)
If possible, what URL is shown? app.destinyitemmanager.com? beta.destinyitemmanager.com?